### PR TITLE
Conditionally load can-debug in development

### PR DESCRIPTION
**Pull request by m-mujica**
Closes donejs#1074

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
   "devDependencies": {
     "coveralls-send": "0.0.2",
     "donejs": "*",
-    "donejs-cli": "^2.0.0-pre.3",
```
Comment (m-mujica) [inline package.json]: we need to merge and release https://github.com/donejs/cli/pull/99 first

Comment (m-mujica) [reply package.json]: I missed this change in my previous PR, maybe this was breaking the build, the generated app tests will fails without steal-conditional being installed.

**Comment by m-mujica** [comment]
@matthewp this is ready for review, too.

**Comment by matthewp** [comment]
lgtm

**Pull request merged**